miasm2/analysis/depgraph.py

```diff
@@ -392,6 +423,20 @@ def __init__(self, ira):
 
         # The IRA graph must be computed
         self._ira.gen_graph()
+        self.cb_follow = [follow_simp_expr, follow_mem_cst]
+
+    def follow_apply_cb(self, expr):
```

Comment:
If this function is public, can you add a `docstring` here ?

Otherwise, you could declare it as protected. In my mind, there is no reason to call this function from outside.